src/services/session.service.js

```diff
@@ -1,34 +1,163 @@
-const CorbadoApi = require('./CorbadoApi');
+const assert = require("assert");
+const jose = require("jose");
+const User = require("../entities/User");
 
 class SessionService {
-    constructor(projectID, apiSecret, apiURL) {
-        this.corbadoApi = new CorbadoApi(projectID, apiSecret,apiURL);
+
+    #version;
+    #shortSessionCookieName;
+    #issuer;
+    #jwksURI;
+    #cacheMaxAge;
+    #client;
+
+    /**
+     * Create a session service.
+     * @param version
+     * @param shortSessionCookieName
+     * @param issuer
+     * @param jwksURI
+     * @param cacheMaxAge
+     * @param client
+     */
+    constructor(version, shortSessionCookieName, issuer, jwksURI, cacheMaxAge, client) {
+        if (!version || !client) {
```

Comment:
when version and client should be required they should be on first argument position.

Reply:
thx